### PR TITLE
Add IDspPreset Interface

### PR DESCRIPTION
**Pull request by TrevorPayne**
Added a new interface for DSP Preset recall

Implemented DSP Preset recall interface in Tesira DSP.

### CODE REVIEW

**Comment by andrew-welker** [comment]
Do we have an issue for this?

**Review by andrew-welker — DISMISSED** [review]

**Comment by andrew-welker** [comment]
Found it: #458

**Comment by TrevorPayne** [comment]
We do have an issue - I wish Auto-Linking was working right.... sigh...

@ndorin I think has issues with this implementation methinks.  We should all discuss sooner rather than later - please revoke the approval if you would.

**Comment by andrew-welker** [comment]
What do you mean by 'it has issues with the implementation'?

**Comment by TrevorPayne** [comment]
What I'm saying is that @ndorin doesn't want it done this way...but we need to have a discussion SOON on how to do this, and not be vague about it.

**Comment by TrevorPayne** [comment]
Updated taking @ndorin suggestions to heart.

File: essentials-framework/Essentials Devices Common/Essentials Devices Common/DSP/DspBase.cs [hunk 1]
```diff
@@ -9,24 +9,34 @@
 
 namespace PepperDash.Essentials.Devices.Common.DSP
 {
-	public abstract class DspBase : EssentialsDevice
+	public abstract class DspBase : EssentialsDevice, IHasDspPresets
```
Comment (andrew-welker) [inline essentials-framework/Essentials Devices Common/Essentials Devices Common/DSP/DspBase.cs:12]: Implementation of `IHasDspPresets` should be on the implementing class, not the base class

**Review by andrew-welker — CHANGES_REQUESTED** [review]

File: essentials-framework/Essentials Devices Common/Essentials Devices Common/DSP/DspBase.cs (hunk 1 above)
Comment (TrevorPayne) [reply essentials-framework/Essentials Devices Common/Essentials Devices Common/DSP/DspBase.cs:12]: I'm going to remove the implementation on the built-in tesira classes.

We should probably consider removing them anyway - my plugin works well and uses appropriate interfaces for essentials.

**Comment by ndorin** [comment]
As @TrevorPayne has suggested, we should remove the Biamp classes from the Essentials framework now that the plugin is available publicly

**Comment by andrew-welker** [comment]
> As @TrevorPayne has suggested, we should remove the Biamp classes from the Essentials framework now that the plugin is available publicly

I think we should do that in a separate PR and make an issue for it.

**Review by andrew-welker — APPROVED** [review]

**Pull request merged**